README.md

```diff
@@ -98,6 +98,25 @@ All directive name should be prepended with `v-`, `v-morph-url` like so.
 
 # Morphling Filters
 
+### morph-age
+
+> `import { morphAge } from 'vue-morphling';`
+
+```html
+<p>{{ new Date(1984, 8, 25) | morph-age }}</p>
+<!-- Numb -->
```

Comment:
@rixlabs can you update this one to the result of the age instead of `<!-- Numb -->`